test/integration.test.js

```diff
@@ -204,10 +205,10 @@ describe('ClientService', () => {
       };
 
       const clientService = await new ClientServiceWithIndexedDb(
-          new ClientService(properties)
+          new ClientService(properties),
       );
       await clientService.registerCertificate();
-      const response = await clientService.validateLedger('non_existing_asset');
+      const response = await clientService.validateLedger('non_existing_asset', 0, 1);
```

Comment:
The validateLedger() interface still accepts the call without specifying asset ages. So, would be better if we could test it anywhere. E.g., using this one or adding the test explicitly.

Reply:
I have added a unit test to test the cases for when the `age` is not specified

Reply:
I think it applies to the node SDK testing as well? (do we have such a test?)

Reply:
Yes, we already have such test for node SDK.